serial_protocol.md

```diff
@@ -0,0 +1,72 @@
+#Protocol:
+During the first use, or when the chiplet changes environments a “NETWORK” call is expected to initialize the wifi parameters.
+
+Every time a serial connection is established we will expect a “INIT” call after which any subsequent calls can be made, until the connection is closed. 
+
+In the following examples we use $ to represent variables. For example $Host represents your firebase host.
+
+##Response Format Byte
+All responses will be prefixed with one of the following bytes signifying the response type.
+```
+  + If response is ok and a raw value
+  * If response is ok and a raw value prefixed by count of bytes in response. 
```

Comment:
We can also use the content-length as an hint to promote to bulk string.

The trick is that for some "growing" value the client code might have to adapt to get predictable output.

Maybe we should have typed GET command where the client can express more intent:
GET$ to force bulk strings
GET: to force numbers
GET+ to force simple strings
GET? to force bool
GET do your best